### DISCUSSION TITLE
Stand up Tuesday May 2nd

### DISCUSSION THREAD

**Discussion by BrandonMagana** [Bugs-io/TC3001B#73] [Main Stand Ups]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 
**What are your priorities for the next 24 hours?**:
 - 
**Blockers**:
 -
**Shoutouts**:
 -@username for x
```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Nothing tbh
 
**What are your priorities for the next 24 hours?**:
- Work on architecture presentation
- Build the dashboard on the front-end
- Finish setting up eslint

**Blockers**:
 - Had a visit :P 

**Shoutouts**:
 - @CristianCazares for his hard work on the front-end

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 - Enjoy the weekend  
 
**What are your priorities for the next 24 hours?**:
 - Set up db on GC
 - Deploy backend 
 
**Blockers**:
 - Lack of will 
 
**Shoutouts**:
 - @CristianCazares for his 21st birthday

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Auth context
 - Protection of routes

**What are your priorities for the next 24 hours?**:
 - Make "upload file button"
 
**Blockers**:
 - none

**Shoutouts**:
 - @BrandonMagana @CarlosCRG19 @Jacobgtz024 for being so handsome
